webextension/background.js

```diff
@@ -117,6 +117,8 @@ class StudyLifeCycleHandler {
       await browser.tabs.create({ url });
     }
 
+    browser.cloudstorage.uninit();
```

Comment:
+1

This ensures it will be called for "expired" and other study-defined endings.